module2/projects/little_shop/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: Little Shop
+length: 1.5 weeks
+tags:
+type: project
+---
+
+## Project Description
+
+You are working for a company developing an E-Commerce Application. Your team is working in a service-oriented architecture, meaning the front and back ends of this application are separate and communicate via APIs. 
+
+Your job is to expose the data that powers the site through an API that the front end would consume. You won't be building an actual front end for this project, so you can look at [these Wireframes](./wireframes) to get an idea of how the front end would integrate with your back end.
```

Comment:
Should we update this verbiage to indicate that they _will_ be connecting this project to a pre-built front end and doing a little fe manipulation after doing so?